rmw_dds_common/include/rmw_dds_common/graph_cache.hpp

```diff
@@ -137,6 +189,7 @@ class GraphCache
     const rmw_qos_profile_t & qos,
     bool is_reader);
 
+
```

Comment:
nitpick: minimize vertical whitespace (https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace)

Reply:
Also just don't make unnecessary diffs - good catch, this was an accidental leftover from development.